platform/darwin/src/MGLStyleLayer.mm.ejs

```diff
@@ -21,9 +21,9 @@ namespace mbgl {
 <% if (layoutProperties.length) { -%>
 <% for (const property of layoutProperties) { -%>
```

Comment:
After the style spec has been processed, the key is removed and inserted into the object as name so I sorted with `_.sortBy(layoutProperties, ['name']`. Unfortunately, it wasn't very well sorted before so the diff increased quite a lot. Divided into separate commits to make it easier to review.